### ISSUE TITLE
Need Client/Project view

### ISSUE THREAD

**Issue by skleinfeldt** [jazkarta/jazkarta.com#5]
Custom view for a Client/Project. Make the layout like this page: http://digital-loom.com/wellesley-college-alumnae-magazine - but don't sweat the styling which will come later. Put Testimonial and Testimonial Source below Description and above "View Website". Say "Download Case Study" instead of "Read Case Study". If optional fields (Design Credit, Case Study) don't exist, don't show them.

Ticket #18 describes the slider at the bottom.

**Comment by cguardia** [comment]
This view is mostly done, but waiting on #15 to put the finishing touches.

**Comment by cguardia** [comment]
Ready. On the beta site now.

**Comment by skleinfeldt** [comment]
Traceback on http://beta.jazkarta.com/projects/my-dante/view - I think it doesn't like the missing testimonial field.

**Comment by skleinfeldt** [comment]
Traceback fixed, 2 more tweaks:

- "Download Case Study" should not show if there is no case study.
- "View Website" should be a link to the Project's URL

**Comment by cguardia** [comment]
Done, but lost the styling on the buttons. Not very sure why.

**Comment by skleinfeldt** [comment]
Fixes look good, will open a separate ticket when we are ready to style the buttons.
